pandas/core/base.py

```diff
@@ -927,16 +927,19 @@ def max(self, axis=None, skipna=True, *args, **kwargs):
         nv.validate_max(args, kwargs)
         return nanops.nanmax(self._values, skipna=skipna)
 
+    @doc(
+        op="max", oppose="min", value="largest",
```

Comment:
I guess this is from black, but I guess you can leave the decorator in a single line, and black will respect it.